### PR TITLE
HWPV-117: add user data to appinsight requests

### PR DESCRIPTION
**Pull request by psoleckimoj**
Based off code here https://github.com/ministryofjustice/use-of-force/pull/599/files

### CODE REVIEW

**Review by hutcheonb-moj — DISMISSED** [review]
LGTM

**Review by hutcheonb-moj — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 1]
```diff
@@ -94,20 +94,15 @@ workflows:
           name: helm_lint
       - hmpps/build_docker:
           name: build_docker
```
Comment (psoleckimoj) [inline .circleci/config.yml:96]: Got fed up of having to approve dev deploys so made it the same as other repos.

**Review by simonnebMOJ — APPROVED** [review]
LGTM!

**Pull request merged**